### ISSUE TITLE
Go to test

### ISSUE THREAD

**Issue by AdrienLemaire** [iamcco/coc-flutter#162]
**Describe the feature**
On VScode, an option "go to test" exists, that will:
- navigate the user to the related test file if exists,
- else create the directory structure and filename, and fill the new file with boilerplate to get started writing the test.

It would be nice to have the option to `:CocCommand flutter.goToTests` and get the same behavior on vim :)

**Comment by dkbast** [comment]
yes this would be amazing - a next logical step would be that renaming/moving one file also results in renaming/moving the other file. This could maybe also be done via a custom_lint rule e.g. "have matching paths and names for test and implementation"'
